app/src/main/java/org/thoughtcrime/securesms/ui/Components.kt

```diff
@@ -89,13 +110,69 @@ fun <T> OptionsCard(card: OptionsCard<T>, callbacks: Callbacks<T>) {
 }
 
 
+@Composable
+fun ItemButton(
+    @StringRes textId: Int,
+    @DrawableRes icon: Int,
+    colors: ButtonColors = transparentButtonColors(),
+    @StringRes contentDescription: Int = textId,
+    onClick: () -> Unit
+) {
+    ItemButton(stringResource(textId), icon, colors, stringResource(contentDescription), onClick)
+}
+
+@Composable
+fun ItemButtonWithDrawable(
+    @StringRes textId: Int,
+    @DrawableRes icon: Int,
+    colors: ButtonColors = transparentButtonColors(),
+    @StringRes contentDescription: Int = textId,
+    onClick: () -> Unit
+) {
+    val context = LocalContext.current
+
+    ItemButton(
+        text = stringResource(textId),
+        icon = {
+            Image(
+                painter = rememberDrawablePainter(drawable = AppCompatResources.getDrawable(context, icon)),
+                contentDescription = stringResource(contentDescription),
```

Comment:
This doesn't seem right. The contentDescription should apply to the button, not the image above. In fact the image should have no content description so that the screen reader reads the button overall.

Reply:
gone